qdrant-landing/content/documentation/concepts/hybrid-queries.md

```diff
@@ -0,0 +1,913 @@
+{
+    "prefetch": {
+        "query": [1, 23, 45, 67], // <------------- small byte vector
+        "using": "mrl_byte"
+        "limit": 1000,
+    },
+    "query": [0.01, 0.299, 0.45, 0.67, ...], // <-- full vector
+    "using": "full",
+    "limit": 10
+}
+```
+
+```python
+from qdrant_client import QdrantClient, models
+
+client = QdrantClient(url="http://localhost:6333")
+
+client.query_points(
+    collection_name="{collection_name}",
+    prefetch=models.Prefetch(
+        query=[1, 23, 45, 67],  # <------------- small byte vector
+        using="mrl_byte",
+        limit=1000,
+    ),
+    query=[0.01, 0.299, 0.45, 0.67, ...],  # <-- full vector
+    using="full",
+    limit=10,
+)
+```
+
+```rust
+use qdrant_client::Qdrant;
+use qdrant_client::qdrant::{PrefetchQueryBuilder, Query, QueryPointsBuilder};
+
+let client = Qdrant::from_url("http://localhost:6334").build()?;
+
+client.query(
+    QueryPointsBuilder::new("{collection_name}")
+        .add_prefetch(PrefetchQueryBuilder::default()
+            .query(Query::new_nearest(vec![1.0, 23.0, 45.0, 67.0]))
+            .using("mlr_byte")
+            .limit(1000u64)
+        )
+        .query(Query::new_nearest(vec![0.01, 0.299, 0.45, 0.67]))
+        .using("full")
+        .limit(10u64)
+).await?;
+```
+
+```java
+import static io.qdrant.client.QueryFactory.nearest;
+
+import io.qdrant.client.QdrantClient;
+import io.qdrant.client.QdrantGrpcClient;
+import io.qdrant.client.grpc.Points.PrefetchQuery;
+import io.qdrant.client.grpc.Points.QueryPoints;
+
+QdrantClient client =
+    new QdrantClient(QdrantGrpcClient.newBuilder("localhost", 6334, false).build());
+
+client
+    .queryAsync(
+        QueryPoints.newBuilder()
+            .setCollectionName("{collection_name}")
+            .addPrefetch(
+                PrefetchQuery.newBuilder()
+                    .setQuery(nearest(1, 23, 45, 67))	// <------------- small byte vector
+                    .setLimit(1000)
+                    .setUsing("mrl_byte")
+                    .build())
+            .setQuery(nearest(0.01f, 0.299f, 0.45f, 0.67f))	 // <-- full vector
+            .setUsing("full")
+            .setLimit(10)
+            .build())
+    .get();
+```
+
+```csharp
+using Qdrant.Client;
+using Qdrant.Client.Grpc;
+
+var client = new QdrantClient("localhost", 6334);
+
+await client.QueryAsync(
+  collectionName: "{collection_name}",
+  prefetch: new List<PrefetchQuery> {
+    new() {
+      Query = new float[] { 1,23, 45, 67 }, // <------------- small byte vector
+        Using = "mrl_byte",
+        Limit = 1000
+    }
+  },
+  query: new float[] { 0.01f, 0.299f, 0.45f, 0.67f }, // <-- full vector
+  usingVector: "full",
+  limit: 10
+);
+```
+
+Fetch 100 results using the default vector, then re-score them using a multi-vector to get the top 10.
+
+```http
+POST /collections/{collection_name}/points/query
+{
+    "prefetch": {
+        "query": [0.01, 0.45, 0.67, ...], // <-- dense vector
+        "limit": 100,
```

Comment:
Is `using` missing?

Reply:
no, the description says it is using the default vector